### PR TITLE
ADEN-8225 Fix not_used status if non_bidder ad is present

### PR DESCRIPTION
**Pull request by nandy-andy**
## Links
* https://wikia-inc.atlassian.net/browse/ADEN-8225

### CODE REVIEW

File: tests/unit/modules/ads/bill-the-lizard-test.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+module('Unit | Module | ads | bill-the-lizard', (hooks) => {
+  hooks.beforeEach(() => {
+    function noop() {}
+
+    window.Wikia.adServices = {
+      billTheLizard: {
+        getPreviousPrediction: noop,
+        getPrediction: noop,
+      },
+      BillTheLizard: {
```
Comment (jacekbj) [inline tests/unit/modules/ads/bill-the-lizard-test.js:13]: Would setting `BillTheLizard` class here (instead of manually created object) work?

Comment (jacekbj) [reply tests/unit/modules/ads/bill-the-lizard-test.js:13]: ditto for `billTheLizard` above

File: app/modules/ads/bill-the-lizard.js [hunk 2]
```diff
@@ -34,10 +36,68 @@ function serializeBids(slotName) {
...
+      // there is a prediction for each other slot than incontent_boxad_1
+        slotStatus = `${BillTheLizard.REUSED};res=${prevPrediction.result};${callId}`;
+      }
+    }
+  }
+
+  return slotStatus;
+}
+
+export default exports = {
```
Comment (jacekbj) [inline app/modules/ads/bill-the-lizard.js]: :gem:

**Review by jacekbj — APPROVED** [review]

File: tests/unit/modules/ads/bill-the-lizard-test.js (hunk 1 above)
Comment (jacekbj) [reply tests/unit/modules/ads/bill-the-lizard-test.js:13]: NVM, I've mistaken import names :)

File: app/modules/ads/bill-the-lizard.js [hunk 3]
```diff
@@ -34,10 +36,68 @@ function serializeBids(slotName) {
...
+      slotStatus = `${BillTheLizard.ON_TIME};res=${result};${callId}`;
+      break;
+    }
+    default: {
+      const prevPrediction = billTheLizard.getPreviousPrediction(
+        incontentsCounter,
+        counter => `incontent_boxad_${counter}`,
+        'cheshirecat',
+      );
+      const bids = exports.getBids();
```
Comment (dmnsobczak) [inline app/modules/ads/bill-the-lizard.js]: There's no need to `getBids()` every cat check if `bids` object is used only for first slot check. Let's rearange `if-else` order and move `getBids()` call inside if body like:
`if (prevPrediction === undefined && exports.getBids()[0])`

Comment (nandy-andy) [reply app/modules/ads/bill-the-lizard.js]: Agree, we can do it better! Although, rearranging will break stuff 🙀 I'll think about it, thanks! 🙇

**Review by dmnsobczak — APPROVED** [review]

File: tests/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+## Console
```
Comment (dmnsobczak) [inline tests/README.md:1]: 💎

**Pull request merged**